radixdlt-java/src/main/java/org/radix/serialization2/client/SerializableObject.java

```diff
@@ -0,0 +1,23 @@
+package org.radix.serialization2.client;
+
+import org.radix.serialization2.DsonOutput;
+import org.radix.serialization2.DsonOutput.Output;
+import org.radix.serialization2.SerializerDummy;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class with serializable fields for version and serializer
+ * that can be inherited to easily implement those fields in
+ * a standard way for DSON and JSON output.
+ */
+public class SerializableObject {
+	@JsonProperty("version")
+	@DsonOutput(Output.ALL)
+	private short version = 100;
```

Comment:
Where should inheriting classes set their version?

Reply:
I decided to go for a minimal approach.  I think simply providing classes with the ability to specify their version number isn't sufficient, or really even necessary for an effective versioning scheme.  This is really a placeholder for a improved approach in my opinion.

In practice, if classes *really* want to override this, they can provide their own `JsonProperty` for "version".